src/extensions/extensionsApi.ts

```diff
@@ -601,6 +601,24 @@ export async function listExtensionVersions(
   return extensionVersions;
 }
 
+/**
+ * @param projectId the project for which we are registering a PublisherProfile
+ * @param publisherId the desired publisher ID
+ */
+export async function getPublisherProfile(
+  projectId: string,
+  publisherId?: string
+): Promise<PublisherProfile> {
+  const res = await api.request("GET", `/${VERSION}/projects/${projectId}/publisherProfile`, {
+    auth: true,
+    origin: api.extensionsOrigin,
+    query: {
+      publisherId,
+    },
+  });
```

Comment:
Please prefer using `apiv2` rather than `api`. It would be great to not add any more usage of `api.request`. Thanks.

Reply:
Done. Wasn't aware of this effort, might be a good idea to mark the old api as deprecated.